app/src/main/java/com/simplecity/amp_library/ui/screens/queue/menu/QueueMenuPresenter.kt

```diff
@@ -31,6 +32,9 @@ class QueueMenuPresenter @Inject constructor(
     albumsRepository,
     navigationEventRelay
 ), QueueMenuContract.Presenter {
+    override fun removeSong(song: Song) {
+        TODO("Not yet implemented")
```

Comment:
Is this future work? If yes there should be an issue opened for it with a link to that issue in a code comment here, if no it needs to be completed in this PR.

Reply:
It's an empty `removeSong` function that needed to be overridden because `Presenter` is being implemented in this class. I think it's not gonna be implemented anytime in the future!